core/src/main/java/fj/data/Validation.java

```diff
@@ -747,7 +748,7 @@ public int hashCode() {
 
 
     public <B, C, D> Validation<List<E>, D> accumulate(Validation<E, B> v2, Validation<E, C> v3, F3<T, B, C, D> f) {
-        List<E> list = fails(List.list(this, v2, v3));
+        List<E> list = fails(list(this, v2, v3));
```

Comment:
@mperry I refactored out some things as and when I touched the class, I don't think it has a effect. I changed this to remove redundant static class reference and used static import there!.